slides/SLEAP.qmd

```diff
@@ -0,0 +1,199 @@
+
+::: aside
+see SLEAP's [Configuring models](https://sleap.ai/guides/choosing-models.html)
+:::
+
+## Start a training job 2/3
+![](img/SLEAP_screenshots/7b_training_centroid.png){fig-align="center" height="500px"}
+
+::: aside
+see SLEAP's [Configuring models](https://sleap.ai/guides/choosing-models.html)
+:::
+
+## Start a training job 3/3
+![](img/SLEAP_screenshots/7c_training_centered_instance.png){fig-align="center" height="500px"}
+
+::: aside
+see SLEAP's [Configuring models](https://sleap.ai/guides/choosing-models.html)
+:::
+
+## Monitor training progress
+![](img/SLEAP_screenshots/7d_training_progress.png){fig-align="center" height="500px"}
+
+# Coffee break ☕ {background-color="#1E1E1E"}
+
+# Practice: SLEAP II {background-color="#03A062"}
+
+## Evaluate trained models
+![](img/SLEAP_screenshots/8_evaluation.png){fig-align="center" height="500px"}
+
+::: aside
+see also the SLEAP [model evaluation notebook](https://sleap.ai/notebooks/Model_evaluation.html)
+:::
+
+## Run inference on new frames
+![](img/SLEAP_screenshots/9_inference.png){fig-align="center" height="400px"}
+
+::: aside
+- Expecte errors on silicon (M1/M2) Macs, see [this discussion](https://github.com/talmolab/sleap/discussions/1151)
+- To correct predictions and update your training data, see SLEAP's  [Prediction-assisted labeling](https://sleap.ai/tutorials/assisted-labeling.html) and [Merging guide](https://sleap.ai/guides/merging.html).
+:::
+
+## Using SLEAP on the HPC cluster
+
+- training and inference are GPU-intensive tasks
+- SLEAP is installed as a module on SWC's HPC cluster
+- `module load sleap`
+- [See this guide for detailed instructions](https://howto.neuroinformatics.dev/data_analysis/HPC-module-SLEAP.html)
+- [Come to the HPC course tomorrow](https://software-skills.neuroinformatics.dev/courses/hpc-behaviour.html)
+
+## Predictions in the sample dataset {.smaller}
+
+`$ cd behav-analysis-course/mouse-EPM/derivatives`
+```{.bash code-line-numbers="false"}
+derivatives/
+├── software-DLC_predictions
+├── software-SLEAP_project
+    └── predictions
+```
+::: {.fragment}
+- Different pose estimation software produce predictions in different formats.
+- Different workflows are needed for importing predicted poses into `Python` for further analysis.
+  - e.g. for `SLEAP` see [Analysis examples](https://sleap.ai/notebooks/Analysis_examples.html)
+:::
+
+## What happens after pose tracking? {.smaller}
+
+:::: {.columns}
+
+::: {.column width="40%"}
+![](img/open-source-tools-fig2a.png)
+:::
+
+::: {.column width="60%"}
+- Load data into Python
+- Visualise and inspect data
+- Clean trajectories:
+  - Identify and drop outliers
+  - Smooth trajectories
+  - Interpolate over missing data
+- Compute variables of interest:
+  - Velocity, acceleration, heading, etc.
+  - Distances/angles between body parts
+  - Time spent in different regions
+  - Application-specific: navigation, social interactions, gait analysis
```

Comment:
This part "overflows" on my screen. Could we make the list shorter or make this slide scrollable?

Reply:
made this slide scrollable